cmd/registry/cmd/compute/vocabulary/vocabulary.go

```diff
@@ -100,6 +100,10 @@ func Command() *cobra.Command {
 			}
 		},
 	}
+	cmd.PersistentFlags().String("filter", "", "Filter selected resources")
+	cmd.PersistentFlags().Bool("dry-run", false, "if set, computation results will only be printed and will not stored in the registry")
+	cmd.PersistentFlags().Int("jobs", 10, "Number of actions to perform concurrently")
```

Comment:
If these aren't here, the flags are unavailable when we instantiate the test command.